validator.js

```diff
@@ -82,6 +82,15 @@
       'pt-PT': /^(\+351)?9[1236]\d{7}$/,
       'el-GR' : /^(\+30)?((2\d{9})|(69\d{8}))$/
     };
+    
+    var currencies = {
+        'USD': /^\$?\-?([0-9]{1,3})?((,[0-9]{3})*|([0-9]{3})*)(\.[0-9]{2})?$/,
+        'CNY': /^¥?\-?([0-9]{1,3})?((,[0-9]{3})*|([0-9]{3})*)(\.[0-9]{2})?$/,
+        'ZAR': /^(R|R )?\-?([0-9]{1,3})?(( [0-9]{3})*|([0-9]{3})*)(,[0-9]{2})?$/,
+        'AUD': /^\$?\-?([0-9]{1,3})?((,[0-9]{3})*|([0-9]{3})*)(\.[0-9]{2})?$/,
+        'HKD': /^\$?\-?([0-9]{1,3})?((,[0-9]{3})*|([0-9]{3})*)(\.[0-9]{2})?$/,
```

Comment:
AUD, HKD and USD have the same regex patterns. Can you pull it out into a variable instead of repeating it?